cookbooks/kinematically_driven_subduction_2d/composition_trms_statistics.cc

```diff
@@ -66,7 +66,7 @@ namespace aspect
             // integral for the whole domain.
             for (unsigned int q = 0; q < n_q_points; ++q)
               {
-                local_total_temperature_square_integral += ((temperature_values[q] * temperature_values[q]) *
+                local_total_temperature_square_integral += (((temperature_values[q]-273.15) * (temperature_values[q]-273.15)) *
```

Comment:
Instead of using 273.15, can you create a `const double kelvin_to_celsius = 273.15` at line 56 (so before the for loop) and use that (4x)? That explains why you subtract that value just from the name :)